rsconf/component/devbox.py

```diff
@@ -96,3 +102,5 @@ def _network(self, jc, z):
         z.ip = n.unchecked_public_ip()
         z.ssh_port = jc.devbox.users[self.user_name]
         n.add_public_tcp_ports([str(z.ssh_port)])
+        z.flask_port = z.ssh_port + 1000
```

Comment:
Why use a different name than SIREPO_PKCLI_SERVICE_PORT? Indeed, it might be easier to generate the export of the vars and have them appended one at a time to jupyter_bashrc.

I think the 1000 should be configurable, just as a matter of principle.

Reply:
Configurable port 9032a86.

If you mean why did I not name it z.service_port I thought the name "service" was too generic and didn't give the reader any indication for what the port is used for. If you mean why did I not name it z.SIREPO_PKCLI_SERVICE_PORT I didn't consider that one. That looks kind of odd to me but is that what you're looking for?

Appending them to jupyter_bashrc sounds like a good idea. Is there a feature in rsconf for appending to a file like that?  Or should I add one (maybe to rsconf.sh)? Or should I build up the string in python and the render in jinja? Or...?

Reply:
z.service_port is in this context understandable, because this is a configurator and it is configuring something that defines that name, no matter how bad it is. The word "flask" doesn't appear anywhere else in the configuration so it is confusing if you are just looking at the configurator component.

Reply:
> Is there a feature in rsconf for appending to a file like that? 

self.rsconf_append

Reply:
https://github.com/radiasoft/rsconf/pull/233/commits/b7d412c9f148cace74a35bd348d4d5437bd03294